utils/install_server.sh

```diff
@@ -73,6 +73,16 @@ if [ "$(id -u)" -ne 0 ] ; then
 	exit 1
 fi
 
+#bail if this system is managed by systemd
+_pid_1_exe="$(readlink -f /proc/1/exe)"
+if [ ${_pid_1_exe##*/} = systemd ]
+then
+	echo "This systems seems to use systemd."
+	echo "Please take a look at the provided example service unit files in this directory, and adapt and install them. Sorry!"
```

Comment:
The two echo … lines should probably write to stderr (`>&2`)

Reply:
I agree, but all the other code in the vicinity also writes fatal error messages to stdout, which is why I just stuck to the accepted current behaviour :)